guide/app_star_catalogue.tex

```diff
@@ -1218,73 +1218,157 @@ \subsection{Designations}
+S  	  	            & S   	& *  & J. South \\
+Rmk 	  	        & RMK 	& ** & C. Rumker \\
+$\Delta$		    & DUN 	& *  & J. Dunlop \\
+Lal  		        & LAL 	& ** & J.-J. de Lalande \\
+O$\Sigma$       	& STT 	& *  & O. von Struve \\
+O$\Sigma\Sigma$ 	& STTA 	& *  & \emph{(Supplemental catalogs)} \\
+BrsO	        	& BSO 	& ** & \emph{Brisbane Observatory} \\
+Jc   	        	& JC  	& *  & W.S. Jacob \\
+Dawes 	        	& DA  	& ** & W. Dawes \\
+Gli 	 	        & GLI 	& ** & J.M. Gilliss \\
+Knott  	        	& KNT 	& ** & G. Knott \\
+Hd, HdO   	        & HDO 	& ** & \emph{Harvard Observatory} \\
+Stone  	       		& STN 	& *  & O. Stone \\
+R   	        	& R   	& *  & H.C. Russell \\
+$\beta$   			& BU  	& *  & S. Burnham \\
+$\beta$ pm 			& BUP 	& *  & \emph{(Proper motion pairs)} \\
+Ho   	        	& HO  	& ** & G.W. Hough \\
+MlbO  	        	& MLO 	& ** & \emph{Melbourne Observatory} \\
+Howe   	        	& HWE 	& ** & H.A. Howe \\
+CapO	        	& CPO 	& ** & \emph{Cape Observatory} \\
+Es 	 	        	& ES  	& ** & Rev. T.E. Espin \\
+CorO	   	     	& COO 	& ** & \emph{Cordoba Observatory} \\
+J   	        	& J   	& *  & R. Jonckheere \\
+Slr   	        	& SLR 	& ** & R.P. Sellors \\
+---		        	& A   	& ** & R. Aitken \\
+---   	        	& I   	& ** & R.T.A. Innes \\
+$\lambda$   	    & SEE 	& *  & T.J.J. See \\
+Hu   	        	& HU  	& ** & W.J. Hussey \\
+$\delta$			& DAW 	& *  & B.H. Dawson \\
+---		        	& B   	& ** & W. van den Bos \\
+---		        	& D   	& ** & H. Dembowski  \\
+Rst   	    	    & RST 	& ** & R.A. Rossiter \\
+$\phi$		    	& FIN 	& ** & W.S. Finsen \\
+Cou 	        	& COU 	& ** & P. Couteau \\
+Mlr					& MLR	& ** & P. Muller \\
+---		        	& HDS  	& ** & \emph{Hipparcos Catalog} \\
+AC		        	& AC  	& ** & A. Clark \\
+AG		        	& AG  	& ** & \emph{Astronomische Gesellschaft Katalog} \\
+AGC		        	& AGC 	& ** & A.G. Clark \\
+Arg		        	& ARG 	& ** & F.W.A. Argelander \\
+All		        	& ALL 	& ** & R.M. Aller \\
+Dju  	        	& DJU 	& ** & P. Djurkovic \\
+Don  	        	& DON 	& ** & H.F. Donner \\
+Doo  	        	& DOO 	& ** & E. Doolittle \\
+Erk  	   			& FRK 	& ** & W.S. Franks \\
+Gale  	    	    & GLE 	& ** & W.F. Gale \\
+Hld   	        	& HLD 	& ** & E.S. Holden \\
+Hrg   		        & HRG 	& ** & L. Hargrave \\
+Ku  		        & KU  	& ** & F. Kustner \\
+Kui  	        	& KUI 	& ** & Gerard P. Kuiper \\
+Pz  	 	        & PZ  	& ** & G. Piazzi \\
+Roe   	        	& ROE 	& ** & E.D. Roe \\
+Se  	 	        & SE  	& ** & A. Secchi \\
+Sei   		        & SEI 	& ** & J. Scheiner \\
+Smyth  	 	        & SMY 	& ** & W.H. Smyth \\
+Vou  	        	& VOU 	& ** & J.G.E.G. Voute \\
+Webb  		        & WEB 	& ** & T.W. Webb \\
+Wnc  	        	& WNC 	& ** & F.A. Winnecke \\
+---		        	& UC  	& ** & \emph{USNO CCD Astrographic Catalog } \\
+Skf		        	& SKF  	& ** & B.A. Skiff \\
+---		        	& TDS, TDT 	& ** & \emph{Tycho Double Star} \\
+Tod		        	& TOI  	& ** & Stassun et al. 		\\
```

Comment:
Typo or real? To**d** vs. TO**I**? And how does Stassun fit to to[di]? (It may be OK, but please double-check!)

Reply:
Real